app/src/ingestion/pdf_postprocess.py

```diff
@@ -114,8 +114,27 @@ def _add_link_markdown(e_text: EnrichedText) -> EnrichedText:
     return e_text
 
 
+def _add_list_markdown(
+    prev_e_text: EnrichedText | None, current_e_text: EnrichedText
+) -> EnrichedText:
+    if (
+        prev_e_text
+        and prev_e_text.type == TextType.LIST_ITEM
+        and current_e_text.type == TextType.LIST_ITEM
+    ):
+        # if sublist, indent 2 spaces
+        if "\u2022" in current_e_text.text:
```

Comment:
Should this be checking for the bullet at the start of the text and not anywhere in the text?

Reply:
there's instances where the bullet appears as the second item due to the paragraph blocks

Reply:
> the second item

Do you mean the second character? If so, use `.strip()`.

Reply:
Add a unit test that shows "where the bullet appears as the second item".

Reply:
by second item I mean something like ` Parent/provider certifications. • Day/date. • Children's names. In/out times. `, Parent/provider certifications is part of the list. I wasn't sure if we should do a find/replace on the  •  values since they'll still be inconsistent (In/out times is another bullet on the same list level)

Reply:
I see. If you're searching for those second bullet items, it makes sense to address them. The current code adds to only the first bullet item: `"  - " + current_e_text.text`. Unit tests would demonstrate the scenarios that you're addressing.